src/components/pages/create-import.js

```diff
@@ -0,0 +1,159 @@
+      documentsToImport: selectedDocs
+    };
+
+    try {
+      const result = await importApiClient.postImportBatch(batch);
+      setIsCreatingNewImportBatch(false);
+      window.location = getImportDetailsUrl(result.batch._id);
+    } catch (error) {
+      handleApiError({ error, t });
+    } finally {
+      setIsCreatingNewImportBatch(false);
+    }
+  };
+
+  useEffect(() => {
+    const getDocuments = async () => {
+      setIsFetchingImportableDocuments(true);
+
+      setImportableDocuments([]);
+      setSelectedDocumentKeys([]);
+
+      try {
+        const { documents } = await importApiClient.getImports(selectedSource.hostName);
+        setImportableDocuments(documents);
+      } catch (error) {
+        handleApiError({ error, t });
+      }
+      setIsFetchingImportableDocuments(false);
+    };
+
+    getDocuments();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [selectedSource]);
+
+  const renderImportType = doc => {
+    let icon = null;
+    if (doc.importType === DOCUMENT_IMPORT_TYPE.add) {
+      icon = <CloudDownloadOutlined />;
+    }
+    if (doc.importType === DOCUMENT_IMPORT_TYPE.update) {
+      icon = <CloudSyncOutlined />;
+    }
+    return <Tooltip title={t(doc.importType)} className="CreateImportPage-importType">{icon}</Tooltip>;
+  };
+
+  const renderTitle = doc => {
+    if (!doc.slug) {
+      return <span>{doc.title}</span>;
+    }
+
+    const baseUrl = getImportSourceBaseUrl(selectedSource);
+
+    const url = `${baseUrl}${getArticleUrl(doc.slug)}`;
+    return <a href={url} target="_blank" rel="noopener noreferrer" >{doc.title}</a>;
+  };
+
+  const renderLanguage = doc => {
+    const languagesData = languageNameProvider.getData(language);
+    const documentLanguageData = languagesData[doc.language];
+    return <CountryFlagAndName code={documentLanguageData.flag} name={documentLanguageData.name} flagOnly />;
+  };
+
+  const renderUpdateDate = doc => {
+    return <span>{formatDate(doc.updatedOn)}</span>;
+  };
+
+  const columns = [
+    { title: t('importType'), key: 'importType', width: '150px', sorter: by(x => x.importType), render: renderImportType },
+    { title: t('title'), key: 'title', sorter: by(x => x.title), render: renderTitle },
+    { title: t('language'), key: 'language', width: '150px', sorter: by(x => x.language), render: renderLanguage },
+    { title: t('updateDate'), key: 'updateDate', width: '200px', sorter: by(x => x.updatedOn), render: renderUpdateDate }
+  ];
+
+  const tableRowSelection = {
+    onChange: selectedRowKeys => {
+      setSelectedDocumentKeys(selectedRowKeys);
+    }
+  };
+
+  return (
+    <Page>
+      <div className="CreateImportPage">
+        <h1>{t('pageNames:import')}</h1>
+
+        <Input
+          placeholder={t('source')}
+          className="CreateImportPage-source"
+          defaultValue={selectedSource?.name}
+          disabled
+          />
+        <br /> <br />
+
+        <Table
```

Comment:
I wanted to use an expandable table here, but the expand button wouldn't appear... So I went the collapse panel variant, not bad but not great either.